src/hyper-loader/Hyper.res

```diff
@@ -279,9 +278,12 @@ let make = (publishableKey, options: option<JSON.t>, analyticsInfo: option<JSON.
         let samsungPayScript = Window.createElement("script")
         samsungPayScript->Window.elementSrc(samsungPayScriptUrl)
         samsungPayScript->Window.elementOnerror(err => {
-          Console.log2("ERROR LOADING SAMSUNG PAY SCRIPT", err)
+          logInfo(Console.log2("ERROR DURING LOADING SAMSUNG PAY SCRIPT", err))
```

Comment:
Is it sending a network request? I believe it should be like 

`logger.setLogInfo(~value="ERROR DURING LOADING GOOGLE PAY SCRIPT", ~eventName=SAMSUNG_PAY_SCRIPT)`

Reply:
Its just a normal console log, if the env is not prod then it will console for better debugging purposes, should this be changed to a log only?

Reply:
Agreed. Can be converted to logApi